pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -578,6 +632,7 @@ func selectVictimsOnNode(
 	pod *v1.Pod,
 	nodeInfo *framework.NodeInfo,
 	pdbs []*policy.PodDisruptionBudget,
+	nodeStatuses *nodeToStatusMap,
```

Comment:
Instead of doing this, lets just return node status instead of the fit boolean. the caller then deals with the status as it sees fit.

Reply:
Done